utils/valgrind_memcheck.supp

```diff
@@ -0,0 +1,588 @@
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds:possible
+   fun:calloc
+   fun:array_alloc_one
+   fun:lrua_array_alloc
+   fun:vos_ts_table_alloc
+   fun:alloc_ts_cache
+   fun:ts_test_init
+   obj:/usr/lib64/libcmocka.so.0.7.0
+   fun:_cmocka_run_group_tests
+   fun:run_ts_tests
+   fun:run_all_tests
+   fun:main
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   ...
+   fun:vea_reserve
+   ...
+   obj:/usr/lib64/libcmocka.so.0.7.0
+   fun:_cmocka_run_group_tests
+   fun:(below main)
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds:reachable
+   ...
+   fun:dl_open_worker
+   ...
+   obj:/usr/lib64/libibverbs.so.1.5.22.1
+}
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds:possible
+   fun:calloc
+   fun:_dl_allocate_tls
+   fun:pthread_create@@GLIBC_2.2.5
```

Comment:
Hardcoded  GLIBC? Is there no easy way?

Reply:
Hi Ravi, thanks for your remarks. Yes, there's another type of wildcard for suppression files, let me check how that would work on this file and send another patch.